Cargo.lock

```diff
@@ -105,15 +105,15 @@ checksum = "5827cebf4670468b8772dd191856768aedcb1b0278a04f989f7766351917b9dc"
 
 [[package]]
 name = "ddprof"
-version = "0.4.0-rc.1"
+version = "0.5.0-rc.1"
 dependencies = [
```

Comment:
Minor: I already merged this change to master in #39 so I think this is going to conflict.